beaker-vue/src/components/panels/FilePanel.vue

```diff
@@ -147,6 +148,7 @@ const emit = defineEmits([
 const props = defineProps([
   "entries",
   "sortby",
+  "viewOnly"
```

Comment:
Instead of a implicit prop setting, better to have an explicit option such as `allowDownloads`, `hideDownload` or even say `disabledColumns` which allows an arbitrary list of strings that match some sort of ident on the columns.